src/app/topics/topics.component.html

```diff
@@ -86,89 +90,62 @@ <h4 class="no_data">
       </div>
 
       <div *ngIf="!dataLoading && filteredProjectsList.length > 0">
-        <div class="d-none d-lg-block">
-          <ng2-smart-table
-            class="table table-striped"
-            [settings]="settings"
-            [source]="filteredProjectsList"
-          ></ng2-smart-table>
-        </div>
-
-        <!-- Mobile -->
```

Comment:
Same as before, the diff is a little hairy here but looks like the code for mobile card-based view was dropped.  Restore!

Reply:
Added back in! Thanks for catching this